### PR TITLE
feat: make fixes to changelog

### PR DESCRIPTION
**Pull request by domeccleston**
Adds smooth scrolling, aligns hero with templates page, adds hover state to links, adds date to changelog item headings

### CODE REVIEW

File: apps/landing/components/changelog/changelog-grid-item.tsx [hunk 1]
```diff
@@ -16,18 +16,27 @@ export async function ChangelogGridItem({ className, changelog }: Props) {
   const baseUrl = process.env.VERCEL_URL
     ? `https://${process.env.VERCEL_URL}`
     : "http://localhost:3000";
+
+  console.log(changelog.date);
```
Comment (perkinsjr) [inline apps/landing/components/changelog/changelog-grid-item.tsx:20]: remove this

File: apps/landing/app/layout.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en" className={`${GeistSans.variable} ${GeistMono.variable}`}>
+    <html lang="en" className={`scroll-smooth ${GeistSans.variable} ${GeistMono.variable}`}>
```
Comment (perkinsjr) [inline apps/landing/app/layout.tsx:38]: Do we want to add scroll smooth to the entire marketing site?

File: apps/landing/components/changelog/side-list.tsx [hunk 3]
```diff
@@ -14,15 +15,16 @@ export function SideList({ logs, className }: SideListProps) {
       <ScrollArea className="h-96 changelog-gradient xl:sticky">
         {logs?.map((changelog, _index) => {
           return (
-            <a
+            <Link
```
Comment (perkinsjr) [inline apps/landing/components/changelog/side-list.tsx:18]: Does scroll still work with a Link ? Prior it did nothing

**Review by perkinsjr — CHANGES_REQUESTED** [review]

File: apps/landing/components/changelog/changelog-grid-item.tsx (hunk 1 above)
Comment (domeccleston) [reply apps/landing/components/changelog/changelog-grid-item.tsx:20]: Done

File: apps/landing/components/changelog/side-list.tsx (hunk 3 above)
Comment (domeccleston) [reply apps/landing/components/changelog/side-list.tsx:18]: Works OK now

File: apps/landing/app/layout.tsx (hunk 2 above)
Comment (domeccleston) [reply apps/landing/app/layout.tsx:38]: discussing

**Review by perkinsjr — APPROVED** [review]

**Pull request merged**